lib/l10n/intl_en.arb

```
@@ -1191,5 +1191,67 @@
+  "saveFailed": "Failed to save logs",
+  "logsSavedTo": "Logs saved to: {path}",
+  "@logsSavedTo": {
+    "placeholders": {
+      "path": {
+        "type": "String"
+      }
+    }
+  },
+  "clearLogsConfirmTitle": "Clear all logs?",
+  "clearLogsConfirmMessage": "This action cannot be undone. All logs will be permanently deleted.",
+  "clear": "Clear",
```

RuboCop (1.81.7)

[warning] Duplicated key in hash literal.

(Lint/DuplicateHashKey)